shared/modules/provider-injection.js

```diff
@@ -77,15 +78,11 @@ function blockedDomainCheck() {
     'blueskybooking.com',
     'sharefile.com',
   ];
-  const currentUrl = window.location.href;
   let currentRegex;
   for (let i = 0; i < blockedDomains.length; i++) {
     const blockedDomain = blockedDomains[i].replace('.', '\\.');
```

Comment:
Re QL alerts: we do a replacement for `.` escaping here so not sure it's actually an issue?